lib/kitchen/directions/bake_toc.rb

```diff
@@ -88,6 +88,11 @@ def self.li_for_page(page)
           number.prepend(child: part_text.paste)
         end
 
+        # Targets the Answer Key
+        if page.has_ancestor?('composite_chapter') && page.ancestor('composite_chapter').search('span').first.text == 'Answer Key'
+          title.first('.os-text').prepend(child: 'Chapter ')
```

Comment:
Currently, 'Chapter ' is a hard-coded string; is there a better way to do this?

Reply:
Use I18n translations.  See [`BakeExample`](https://github.com/openstax/kitchen/blob/master/lib/kitchen/directions/bake_example.rb#L16) which pulls in the translation for the `example_label` key in [`lib/locales/en.yml`](https://github.com/openstax/kitchen/blob/master/lib/locales/en.yml#L5).

Reply:
I'm wondering if Chapter is added in the baked version you have, I wonder if it is also in the answer key page title and can just be pulled in that way.

Reply:
Yes, it looks like Chapter is added in the answer key page, and that it would work to just pull that into the ToC. I don't see a `BakeAnswerKey` class, so I'm going to go ahead and try to get one working, and you can tell me when you have a chance if I'm barking up the wrong tree 😅